test/manifold_test.cpp

```diff
@@ -711,4 +711,28 @@ TEST(Manifold, PinchedVert) {
   EXPECT_FALSE(touch.IsEmpty());
   EXPECT_EQ(touch.Status(), Manifold::Error::NoError);
   EXPECT_EQ(touch.Genus(), 0);
-}
\ No newline at end of file
+}
+
+TEST(Manifold, Hull) {
+#ifdef MANIFOLD_EXPORT
```

Comment:
This should only wrap `ExportMesh` or else the test often won't run.

Reply:
Ah I just totally forgot about those ifdefs. Fixed, since the few test lines are about the only things that are going to survive from these initial commits (replace with a vendored algorithm).